sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -450,6 +432,13 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
     prepareRenderFrame(creatingSurface = true)
   }
 
+  @MainThread
+  fun setScreenRefreshRate(refreshRate: Float) {
```

Comment:
any reasons for `float` parameter?

Reply:
cause Android SDK actually returns `float`, see https://developer.android.com/reference/android/view/Display#getRefreshRate()
we use int later then, just because it's a bit easier in calculation